LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright (C) 2019 The Regents of the University of California.
```

Comment:
i think these should be 2018 -- probably earlier but unfortunately i didn't save the commit history when i ported the code from data concierge.

Reply:
https://github.com/CAIDA/pywandio/commit/8ebc34828477e57dcdebb2d66cb0cba77fedc9fe

Reply:
Updated `LICENSE` and `debian/copyright` dates in e7c795b